projects/plugins/jetpack/composer.json

```diff
@@ -32,6 +32,7 @@
 		"automattic/jetpack-logo": "1.5.x-dev",
 		"automattic/jetpack-options": "1.13.x-dev",
 		"automattic/jetpack-partner": "1.5.x-dev",
+		"automattic/jetpack-post-list": "0.1.x-dev",
```

Comment:
Just a note to remove this before we merge.

Reply:
removing...